frontend/pages/index.js

```diff
@@ -185,22 +177,22 @@ const getOrganizations = async token => {
   }
 };
 
-const parseProjects = projects => {
+const parseProjects = (projects) => {
   console.log("Parsing projects...");
```

Comment:
I think this console.log can also be removed, I think I missed this in my previous review